units/ArmBots/armpw.lua

```diff
@@ -124,6 +124,7 @@ return {
 				size = 1.75,
 				soundhitwet = "splshbig",
 				soundstart = "flashemg",
+				soundtrigger = true,
```

Comment:
Doesn't look like a banshee

Reply:
This pr adds fixes to armpw/armbrawl/armflash too, i should note that in the desc

Reply:
What I'm saying is that adding soundTrigger to a bunch of units sounds unrelated to changing Banshee, and should probably be reviewed by different people. Why not make two PRs, one for soundTrigger and one for Banshee?